utils/loggers/wandb/wandb_utils.py

```diff
@@ -318,77 +399,123 @@ def log_model(self, path, opt, epoch, fitness_score, best_model=False):
         fitness_score (float) -- fitness score for current epoch
         best_model (boolean) -- Boolean representing if the current checkpoint is the best yet.
         """
-        model_artifact = wandb.Artifact('run_' + wandb.run.id + '_model',
-                                        type='model',
-                                        metadata={
-                                            'original_url': str(path),
-                                            'epochs_trained': epoch + 1,
-                                            'save period': opt.save_period,
-                                            'project': opt.project,
-                                            'total_epochs': opt.epochs,
-                                            'fitness_score': fitness_score})
-        model_artifact.add_file(str(path / 'last.pt'), name='last.pt')
-        wandb.log_artifact(model_artifact,
-                           aliases=['latest', 'last', 'epoch ' + str(self.current_epoch), 'best' if best_model else ''])
+        model_artifact = wandb.Artifact(
+            "run_" + wandb.run.id + "_model",
+            type="model",
+            metadata={
+                "original_url": str(path),
+                "epochs_trained": epoch + 1,
+                "save period": opt.save_period,
+                "project": opt.project,
+                "total_epochs": opt.epochs,
+                "fitness_score": fitness_score,
+            },
+        )
+        model_artifact.add_file(str(path / "last.pt"), name="last.pt")
+        wandb.log_artifact(
+            model_artifact,
+            aliases=[
+                "latest",
+                "last",
+                "epoch " + str(self.current_epoch),
+                "best" if best_model else "",
+            ],
+        )
         LOGGER.info(f"Saving model artifact on epoch {epoch + 1}")
 
-    def log_dataset_artifact(self, data_file, single_cls, project, overwrite_config=False):
+    def log_dataset_artifact(
+        self,
+        data_file,
+        single_cls,
+        entity,
+        project,
+        alias="latest",
+        overwrite_config=False,
+    ):
         """
         Log the dataset as W&B artifact and return the new data file with W&B links
 
         arguments:
         data_file (str) -- the .yaml file with information about the dataset like - path, classes etc.
         single_class (boolean)  -- train multi-class data as single-class
+        entity (str) -- entity name. Used to construct the artifact path
```

Comment:
We need to pass the entity and the alias for the artifact so that the wandb.config `_sanitize` function doesn't fail when it sees an artifact value being logged.